scripts/ci_diff_coverage.rb

```
@@ -0,0 +1,180 @@
+#!/usr/bin/env ruby
```

RuboCop (1.84.0)

[warning] Script file ci_diff_coverage.rb doesn't have execute permission.

(Lint/ScriptPermission)